queries.sql

```diff
@@ -72,4 +72,48 @@ GROUP BY neutered;
 select species, min(weight_kg) as minimum, max(weight_kg) as maximum from animals 
 GROUP BY species
 select species, avg(escape_attempts) from animals 
-where extract(YEAR FROM date_of_birth) BETWEEN 1990 AND 2000 GROUP BY species
\ No newline at end of file
+where extract(YEAR FROM date_of_birth) BETWEEN 1990 AND 2000 GROUP BY species
+
+/*Queries with JOINs*/
+-- What animals belong to Melody Pond?
+SELECT A.name, A.date_of_birth, A.escape_attempts, A.neutered, A.weight_kg 
+FROM animals A
+JOIN owners OW ON A.owner_id = OW.id
+WHERE OW.full_name = 'Melody Pond';
+
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT A.name, A.date_of_birth, A.escape_attempts, A.neutered, A.weight_kg, S.name as Species
```

Comment:
- [ ] Instead of writing ` A.name, A.date_of_birth, A.escape_attempts, A.neutered, A.weight_kg` .you can use `*` like 
```
SELECT * FROM animals

```